ans2ask-ui/src/components/Question/Question.jsx

```diff
@@ -4,11 +4,12 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faUser } from "@fortawesome/free-solid-svg-icons";
 import { useNavigate } from "react-router-dom";
 import { url } from "../../utils/Constants.jsx";
+import Highlighter from "react-highlight-words";
 import "./Question.css";
 
-const MAX_LENGTH = 370;
+const MAX_LENGTH = 369;
```

Comment:
why we are changing this?

Reply:
I made some changes to the question card and the text overflowed so I had to decrease the max length a bit to fix it